### PR TITLE
Set up routing for judging

### LINKED ISSUE #62: Judging routing
Set up subpages for the judging page

Pages for judging include:
- judging/
- judging/view?project={id}
- judging/done <- maybe

### PR DESCRIPTION
**Pull request by ianmah**
Closes #62 

Creates new routes 
- `/judging` 
- `/judging/view/:id`
- `/judging/done` 
- 404 page

Only includes JUDGING in sidebar when judging is open

### CODE REVIEW

File: src/components/Page.js [hunk 1]
```diff
@@ -22,9 +23,16 @@ const Content = styled.div`
 
 const Page = ({ children }) => {
   const [showMobileSidebar, setShowMobileSidebar] = useState(false)
+  const [isJudgingEnabled, setIsJudgingEnabled] = useState(false)
+
+  useEffect(() => {
+    const unsubscribe = isJudgingOpen(setIsJudgingEnabled)
```
Comment (jackyzha0) [inline src/components/Page.js:29]: why the name change from isJudgingOpen to isJudgingDisabled?

File: src/pages/Judging/Done.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+export default () => (
+  <h1>
+    Done!
+  </h1>
+)
```
Comment (jackyzha0) [inline src/pages/Judging/Done.js:7]: we should probably add // TODO: impl or something of the like for the stub pages

File: src/components/Page.js (hunk 1 above)
Comment (ianmah) [reply src/components/Page.js:29]: idk its overlapping with the firebase utility function name

File: src/components/Page.js [hunk 3]
```diff
@@ -1,7 +1,8 @@
-import React, { useState } from 'react'
+import React, { useState, useEffect } from 'react'
 import styled from 'styled-components'
 import Sidebar from './Sidebar'
 import MobileMenuBar from './MobileMenuBar'
+import { isJudgingOpen } from '../utility/firebase'
```
Comment (jackyzha0) [inline src/components/Page.js:5]: alternatively, we can do import as here

**Review by jackyzha0 — APPROVED** [review]
🧼

**Pull request merged**